lib/core/shell/step.sh

```diff
@@ -214,41 +265,3 @@ function re_source() {
   exec 2>&3
   exec 1>&4
 }
-
-function ih::setup::core.shell::private::validate-profile() {
```

Comment:
Removing this because `collect-env-var` is checking whether the new env vars are non-empty.

Reply:
But it's called up on line 19 to discover whether we need to run this step at all...